modules/reissue-internal-certificates-secured-clusters-automatic-upgrades.adoc

```diff
@@ -16,12 +21,12 @@ You can reissue internal certificates for Sensor, Collector, and Admission Contr
 
 . In the RHACS portal, navigate to *Platform Configuration* -> *Clusters*.
 . In the *Clusters* view, select a *Cluster* to view its details.
-. From the cluster details panel, select the link to *apply credentials by using an automatic upgrade*.
+. From the cluster details panel, select the link to *Apply credentials by using an automatic upgrade*.
 
 [NOTE]
 ====
 When you apply an automatic upgrade, {product-title} creates new credentials in the selected cluster. However, you will still see a notification. The notification goes away when each {product-title} service begins using the new credentials after the service restarts.
 ====
 
-//.Additional resources
-//* TODO: Add link to the automatic upgrades topic
+.Additional resources
+* xref:../installing/install-quick-roxctl.adoc#installing-central[Installing central]
```

Comment:
These lines need to be removed from the module and added to the assembly right under the include for this module. Also needs the add'l resources role metadata tag.